.github/workflows/ci.yml

```diff
@@ -93,22 +93,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        base-image: [ alpine ]
+        include:
+          - base-image: alpine
+            os-type: linux-musl
+          - base-image: centos
+            os-type: linux-glibc
     runs-on: ubuntu-20.04
     timeout-minutes: 60
     steps:
     - uses: actions/checkout@v3.3.0
     - name: Build in Docker container
       run: |
         set -e
-        docker build -t mybuildimage -f "./docker/${{ matrix.base-image }}.dockerfile" .
-        docker run -e OS_TYPE=linux-musl --rm --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project mybuildimage \
+        docker build -t mybuildimage -f "./docker/${{ matrix.base-image }}.dockerfile" ./docker
+        docker run -e OS_TYPE=${{ matrix.os-type }} --rm --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project mybuildimage \
           ./build.sh Workflow --containers none
     - name: Test the Shell scripts from README.md in Docker container
       run: |
         set -e
-        docker build -t mybuildimage -f "./docker/${{ matrix.base-image }}.dockerfile" .
-        docker run -e OS_TYPE=linux-musl --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project --rm mybuildimage /bin/sh -c '
+        docker build -t mybuildimage -f "./docker/${{ matrix.base-image }}.dockerfile" ./docker
+        docker run -e OS_TYPE=${{ matrix.os-type }} --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project --rm mybuildimage /bin/sh -c '
```

Comment:
```suggestion
        docker run -e --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project --rm mybuildimage /bin/sh -c '
```

Reply:
OS_TYPE should be not needed here

Reply:
removed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2193/commits/8ec9efa9cc818523f030d9c2606bb614f6650d71